src/software/ai/hl/stp/tactic/dribble/dribble_fsm.cpp

```diff
@@ -106,6 +136,33 @@ void DribbleFSM::dribble(const Update &event)
             event.control_params.dribble_destination,
             event.control_params.final_dribble_orientation);
 
+    Point dribble_destination = getDribbleBallDestination(
+        event.common.world.ball().position(), event.control_params.dribble_destination);
+
+    // when we have the ball in the dribbler, only then we pivot around the ball
+    // if (distance(event.common.robot.position(), event.common.world.ball().position()) <
+    // 0.088)
+    if (event.common.robot.isNearDribbler(event.common.world.ball().position()))
+    {
+        Angle ang_1 = event.common.robot.orientation() + Angle::fromDegrees(40);
+        Angle ang_2 = event.common.robot.orientation() - Angle::fromDegrees(40);
+        // find the closer angle to the target orientation
+        Angle target_angle = (ang_1 - target_orientation).clamp().abs() <
+                                     (ang_2 - target_orientation).clamp().abs()
```

Comment:
FYI, You can use `Angle::minDiff` instead since it does the same thing as getting the absolute value of clamp.